src/PointwiseFunctions/AnalyticData/GrMhd/CylindricalBlastWave.hpp

```diff
@@ -220,24 +216,17 @@ class CylindricalBlastWave : public MarkAsAnalyticData {
   void pup(PUP::er& /*p*/) noexcept;  //  NOLINT
 
  private:
-  InnerRadius::type inner_radius_ =
-      std::numeric_limits<double>::signaling_NaN();
-  OuterRadius::type outer_radius_ =
-      std::numeric_limits<double>::signaling_NaN();
-  InnerDensity::type inner_density_ =
-      std::numeric_limits<double>::signaling_NaN();
-  OuterDensity::type outer_density_ =
-      std::numeric_limits<double>::signaling_NaN();
-  InnerPressure::type inner_pressure_ =
-      std::numeric_limits<double>::signaling_NaN();
-  OuterPressure::type outer_pressure_ =
-      std::numeric_limits<double>::signaling_NaN();
-  MagneticField::type magnetic_field_ =
-      std::array<double, 3>{{std::numeric_limits<double>::signaling_NaN(),
-                             std::numeric_limits<double>::signaling_NaN(),
-                             std::numeric_limits<double>::signaling_NaN()}};
-  AdiabaticIndex::type adiabatic_index_ =
-      std::numeric_limits<double>::signaling_NaN();
+  double inner_radius_ = std::numeric_limits<double>::signaling_NaN();
+  double outer_radius_ = std::numeric_limits<double>::signaling_NaN();
+  double inner_density_ = std::numeric_limits<double>::signaling_NaN();
+  double outer_density_ = std::numeric_limits<double>::signaling_NaN();
+  double inner_pressure_ = std::numeric_limits<double>::signaling_NaN();
+  double outer_pressure_ = std::numeric_limits<double>::signaling_NaN();
+  std::array<double, 3> magnetic_field_{
+      {std::numeric_limits<double>::signaling_NaN(),
+       std::numeric_limits<double>::signaling_NaN(),
+       std::numeric_limits<double>::signaling_NaN()}};
```

Comment:
Should be cleaner as `auto blah = make_array<3>(nan)`, also applies throughout the PR.

Reply:
I chose this way (i.e explicit declaration of a size 3 array over make_array) to make these declarations consistent to eliminate including make_arry; but I don't have a strong preference

Reply:
Ah, okay; thanks for clarifying